core/templates/dev/head/components/summary_tile/collection_summary_tile_directive.html

```diff
@@ -37,13 +30,14 @@ <h3 class="activity-title"><[getCollectionTitle() || DEFAULT_EMPTY_TITLE]></h3>
   </md-card>
 </script>
 
+
 <style>
   .oppia-activity-summary-tile .thumbnail-image {
-    top: 15.5%;
+      top: 15.5%;
```

Comment:
Not sure why the indentation here (and below) has changed. It was correct before.